[@types/dragula] Implemented slideFactorX and slideFactorY parameters for enhanced dragging control

ValerioEmanuele opened the pull request:
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).


If changing an existing definition:

- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<https://www.npmjs.com/package/dragula>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the `package.json`.

andrewbranch commented on package.json:
```diff
@@ -1,7 +1,7 @@
     "private": true,
     "name": "definitely-typed",
-    "version": "0.0.3",
+    "version": "0.0.4",
```
Please revert this change. It’s not related to the @types packages inside the repo.

```suggestion
    "version": "0.0.3",
```

andrewbranch reviewed (changes requested):
Looks good except for the package.json change

andrewbranch commented on types/dragula/v2/index.d.ts:
```diff
@@ -17,6 +17,8 @@ declare namespace dragula {
         delay?: boolean | number | undefined;
         mirrorContainer?: Element | undefined;
         ignoreInputTextSelection?: boolean | undefined;
+        slideFactorX?: number | undefined;
+        slideFactorY?: number | undefined;
```
It doesn’t look like these exist in v2: https://github.com/bevacqua/dragula/tree/v2.1.2

ValerioEmanuele replied:
removed

ValerioEmanuele commented:
Hi @andrewbranch ,

I've addressed the feedback you provided:

1. Regarding your comment on the properties not existing in v2 ("It doesn’t look like these exist in v2: https://github.com/bevacqua/dragula/tree/v2.1.2"), I have removed those properties from v2 as suggested.
2. Concerning your note about the package.json change ("Looks good except for the package.json change"), I have reverted it to the previous version.

Please let me know if there's anything else that needs to be adjusted.

Thanks!

andrewbranch reviewed (approved):
Thanks!

ValerioEmanuele commented:
Ready to merge
